samtranslator/internal/schema_source/aws_serverless_statemachine.py

```diff
@@ -174,6 +174,7 @@ class Properties(BaseModel):
     Type: Optional[PassThroughProp] = properties("Type")
     AutoPublishAlias: Optional[PassThroughProp]
     DeploymentPreference: Optional[PassThroughProp]
+    UseAliasAsEventTarget: Optional[bool]
```

Comment:
```suggestion
    UseAliasAsEventTarget: Optional[bool] # TODO: add docs
```
Let's add TODO to make sure we add the doc here in the future.

Reply:
I am working on it, so I won't add a TODO here.